client/dashboard/app/router.tsx

```diff
@@ -62,8 +62,8 @@ const sitesRoute = createRoute( {
 
 const siteRoute = createRoute( {
 	getParentRoute: () => rootRoute,
-	path: 'sites/$siteId',
-	loader: ( { params: { siteId } } ) => maybeAwaitFetch( siteQuery( siteId ) ),
+	path: 'sites/$siteSlug',
```

Comment:
Isn't this just the domain? Why not call it domain?

Reply:
Hmm.... good question. Maybe it's about the context. I observe that in the codebase, we say "domain" when we're talking about **domain management**, e.g. in https://wordpress.com/domains/manage.

But in the context of **site management**, internally we usually refer to this as "site slug". Basically it is a domain which is:

- the primary domain of the site
- the one that's displayed in the URL
- the one that's shown in the site selector, etc.

Reply:
This is super confusing to me because `slug` is normally the parts of the URL that comes after the domain. In all cases this would be a domain, no?

Reply:
Yes, it is a domain, but since a site can have multiple domains, we designate "slug" as the one that's we primarily using to refer to the site (in URL fragments and in dashboard(s)).

We are already using this term (site slug) everywhere in wpcom backend, Jetpack mu-plugins, onboarding code, etc... so I'd like to keep this consistency and not introduce any divergence.

Reply:
ok